dataclients/kubernetes/endpointslices.go

```diff
@@ -125,6 +127,10 @@ type EndpointSliceEndpoints struct {
 	// https://kubernetes.io/docs/concepts/services-networking/topology-aware-routing/#safeguards
 	// Zone aware routing will be available if https://github.com/zalando/skipper/issues/1446 is closed.
 	Zone string `json:"zone"` // "eu-central-1c"
+	// Node hosting this endpoint. This can be used to determine endpoints local to a node.
+	NodeName string `json:"nodeName"`
+	// TargetRef is a reference to a Kubernetes object that represents this endpoint.
+	TargetRef *objectReference `json:"targetRef"`
```

Comment:
maybe `PodName string` is enough here
Namespace we get from endpointslice metadata

Reply:
This is just a datastruct to get data from k8s. We can decide what we use in the metadata route.

Reply:
yes here true but not at https://github.com/zalando/skipper/pull/2845/files#r1450668487